testing/vcs/test_vcs_set_cmap_colorcell.py

```diff
@@ -0,0 +1,10 @@
+import vcs
```

Comment:
@doutriaux1 like I said, we are having two many tests. In this case, I think you can easily put both of them together into one as they are testing getter and setters.